doc/readthedocs/problemtypes.rst

```diff
@@ -161,14 +161,16 @@ Particle
 
 **General**
 
-This is a completely different problem type, which comes along without elements.
-Instead, the method is based on SPH, which means *smoothed particle hydrodynamics*,
-and the particles are defined by their coordinates.
+This is a completely different discretization type that comes along without elements but uses particles, i.e., spatial discretization points.
```

Comment:
```suggestion
This is a completely different discretization type without the typical mesh consisting of elements.
Instead, this mesh-free approach uses particles that can move in space.
```

I think this is what you try to say by "spatial discretization points"?

Reply:
I prefer to leave the description as I wrote it.

Reply:
Edit: I changed the description to be more precise.